File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.c
int removeDuplicates(int* nums, int numsSize){
    int i;
    
    int j = 1;
    for(i = 1; i < numsSize; i++)
    {
        if(nums[i] != nums[i-1])
            nums[j++] = nums[i];
    }
    return j;
}

File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int val = nums[0];
        int k = 1;
        
        for(int i = 1; i < nums.length; i++){
            if(nums[i] != val){
                val = nums[i];
                nums[k++] = nums[i]; 
            }
        }
        
        return k;
    }
}